### ISSUE TITLE
Fix timeline list to be more accessible per accessibility guidelines

### ISSUE THREAD

**Issue by kamerynB** [project-lux/lux-frontend#865]
**Problem Description:**
The timeline list is difficult for keyboard users to navigate.

**Expected Behavior/Solution:**
The timeline list should be accessible by tabbing. Once focused on the timeline, a user can then use the keyboard arrows to navigate each list element and link.

**Requirements:**

- [ ] The timeline list will be a focusable element via tabbing
- [ ] The timeline list rows and links will be accessible via the keyboard arrows

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Wireframe/Mockup - Mike
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Are new regression tests required for QA - Amy
- [ ] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**

- https://lux-front-tst.collections.yale.edu/view/person/66049111-383e-4526-9632-2e9b6b6302dd

**Dependencies/Blocks:**
N/A

**Related Github Issues:**

- META: https://github.com/project-lux/lux-frontend/issues/487

**Related links:**
N/A

**Wireframe/Mockup:**
TBD

**Comment by kamerynB** [comment]
Update 7/6:
After some development and code review, it was decided to reach back out to Nick in accessibility to get clarification on requirements. This will ensure we are adhering to Yale accessibility requirements.

**Comment by zachjschwartz** [comment]
Deployed to Prod. Closing.
